flytekit/image_spec/default_builder.py

```diff
@@ -476,7 +476,6 @@ def _build_image(self, image_spec: ImageSpec, *, push: bool = True) -> str:
 
             command = [
                 "docker",
-                "image",
```

Comment:
@thomasjpfan is it okay to remove this?

Reply:
It should be okay. When I run `docker build --help`, it's an alias for `docker image build`:

```bash
❯ docker build --help
Start a build

Usage:  docker buildx build [OPTIONS] PATH | URL | -

Start a build

Aliases:
  docker build, docker builder build, docker image build, docker buildx b
```

All the options we use, i.e. `--tag`, `--platform`, `--secret` , `--push` are supported based on `docker build --help`.

Reply:
nice, going to merge it, thnaks you @ttitsworth-lila